templates/Program/program.html.twig

```diff
@@ -246,19 +246,25 @@
   ></div>
   <div class="js-project-description-credits"
        data-project-id="{{ program.id }}"
+       data-max-length={{ max_description_size }}
+       data-has-description={% if program.description %} true {% else %} false {% endif %}
+       data-has-credits={% if program.credits %} true {% else %} false {% endif %}
        data-trans-more-info="{{ "more-information"|trans({}, "catroweb") }}"
        data-trans-less-info="{{ "less-information"|trans({}, "catroweb") }}"
        data-trans-description="{{ "description"|trans({}, "catroweb") }}"
        data-trans-notes-and-credits="{{ "credits"|trans({}, "catroweb") }}"
        data-trans-translation-saved="{{ "programs.translationSaved"|trans({}, "catroweb") }}"
        data-trans-translation-deleted="{{ "programs.translationDeleted"|trans({}, "catroweb") }}"
-       data-trans-close-editor="{{ "programs.closeEditorPrompt"|trans({}, "catroweb") }}"
+       data-trans-close-description-editor="{{ "programs.closeDescriptionEditor"|trans({}, "catroweb") }}"
+       data-trans-close-credits-editor="{{ "programs.closeCreditsEditor"|trans({}, "catroweb") }}"
+       data-trans-close-editor-prompt="{{ "programs.closeEditorPrompt"|trans({}, "catroweb") }}"
        data-trans-save="{{ "programs.saveDescription"|trans({}, "catroweb") }}"
```

Comment:
I think this should be updated to the new `save` translation. Then the `saveDescription` and `saveCredits` can probably be removed since they are not used anymore